lingocore/lib/screens/course_screen.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+
+class CourseScreen extends StatefulWidget {
+  const CourseScreen({super.key});
+
+  @override
+  State<CourseScreen> createState() => _CourseScreenState();
+}
+
+class _CourseScreenState extends State<CourseScreen> {
+  
+  final List<String> lessons = List.generate(20, (index) => 'Ders ${index + 1}');
+  final List<String> lessonDescription = List.generate(20, (index) => 'Bu ${index + 1}. dersin içeriğidir.');
+  
+  
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // to prevent the application from slowing down as the number of lessons increases.
+      body: ListView.builder(
+        itemCount: lessons.length,
+        itemBuilder: (context, index) {
+
+          final lessonTitle = lessons[index];
+          final lessonsDescription = lessonDescription[index];
+
+          return Container(
+            // width: 300,
+            // height: 100,
+            padding: EdgeInsets.all(50),
+            margin: EdgeInsets.all(100),
```

Comment:
Margin kullanmak yerine padding ile sar.

Reply:
Büyük paddingler küçük ekranlarda layoutun bozulmasına neden oluyor.
Responsive tasarım için center ve maksimum boyut kullan.